Makefile

```diff
@@ -270,17 +272,25 @@ BUNDLE_IMGS ?= $(BUNDLE_IMG)
 # The image tag given to the resulting catalog image (e.g. make catalog-build CATALOG_IMG=example.com/operator-catalog:v0.2.0).
 CATALOG_IMG ?= $(IMAGE_TAG_BASE)-catalog:v$(VERSION)
 
-# Set CATALOG_BASE_IMG to an existing catalog image tag to add $BUNDLE_IMGS to that image.
-ifneq ($(origin CATALOG_BASE_IMG), undefined)
-FROM_INDEX_OPT := --from-index $(CATALOG_BASE_IMG)
-endif
-
-# Build a catalog image by adding bundle images to an empty catalog using the operator package manager tool, 'opm'.
-# This recipe invokes 'opm' in 'semver' bundle add mode. For more information on add modes, see:
+# Build a file based catalog image using opm.
+# https://docs.openshift.com/container-platform/4.12/operators/admin/olm-managing-custom-catalogs.html
 # https://github.com/operator-framework/community-operators/blob/7f1438c/docs/packaging-operator.md#updating-your-existing-operator
 .PHONY: catalog-build
-catalog-build: opm ## Build a catalog image.
-	$(OPM) index add --container-tool docker --mode semver --tag $(CATALOG_IMG) --bundles $(BUNDLE_IMGS) $(FROM_INDEX_OPT)
+catalog-build: opm
+	@[ ! -d catalog ] && mkdir -p catalog || true
+	@[ ! -f catalog.Dockerfile ] && $(OPM) generate dockerfile catalog || true
+	$(OPM) init  sandboxed-containers-operator --default-channel=$(DEFAULT_CHANNEL) --description ./README.md --output yaml > catalog/index.yaml
```

Comment:
This should be done once and then checked in to the repo.  We need static files that we can modify as needed to make thing clear.

Reply:
If we are to do check in generated files, then I'd rather not have them created as part of the main build flow.

Reply:
catalog-init is not in the main build flow.  Normal is make catalog-build; make catalog-push

If you are using a private quay.io, the catalog files need to be re-written.  make catalog-clean; make catalog-init will do that for you.